packages/render/rollup.config.js

```diff
@@ -36,12 +38,4 @@ const config = Object.assign({}, configBase, {
   ],
 });
 
-const prodConfig = Object.assign({}, configBase, {
```

Comment:
@jeetiss maybe we should remove the min versions in all of the packages?

Reply:
Yep, we can do it, but it has the problem as with ESM bundle. Truly this is a breaking change for a package, but I think that we can write the sentence about it in a changelog and everything should be okay because nobody uses it

Reply:
We can remove minified bundles in other packages as separate PRs. Let me know how you prefer proceeding 😄 Regarding the major bump, I'm more than ok on proceeding with major! I definitely see your point.